dask_sql/context.py

```diff
@@ -250,6 +250,13 @@ def create_table(
             **kwargs,
         )
 
+        if type(input_table) == str:
+            filepath = input_table
+            dc.filepath = input_table
+            self.schema[schema_name].filepaths[table_name.lower()] = input_table
+        else:
+            filepath = ""
```

Comment:
Using empty string here as `None` throws an error.

Reply:
I'm fine with `""` but wonder why `None` was causing issues? hmm

Reply:
What the error on the Rust side or Python side?

Reply:
Ahh, yep. That is worth doing I think. Thanks for pointing that out Charles.

Reply:
Thanks @charlesbluca for looking into this!